data/_ui/tooltips.txt

```diff
@@ -1092,23 +1092,242 @@ tip "This outfit is considered an atrocity."
 
 
 # Controls
-tip `Reverse`
-	`Turn opposite to the flight direction (or use reverse engines if you have them installed). Combine with <shift> key to stop your ship.`
+tip "Forward thrust"
+	`Accelerate your flagship forward.`
+
+tip "Turn left"
+	`Turn your flagship left.`
+
+tip "Turn right"
+	`Turn your flagship right.`
+
+tip "Reverse"
+	`Turn your flagship opposite its current direction of flight. If reverse thrusters are installed, accelerate backwards. With <shift>, stop your ship.`
+
+tip "Fire afterburner"
+	`Fire your flagship's afterburner, if one is installed.`
+
+tip "Auto steer"
+	`If the "Automatic aiming" setting is off, active automatic aiming when this key is pressed.`
+
+tip "Land on planet / station"
+	`Land on the selected planet. If no planet is selected, automatically selects a planet to land on.`
```

Comment:
Should we comment on the double press switching between planets?

Reply:
```suggestion
	`Land on the selected planet. If no planet is selected, automatically selects a planet to land on. If a planet is already selected, double tapping will cycle to the next planet.`
```

Reply:
Oh and also mention that pressing it while flying over a planet will try to land on that planet, even if it's not landable.

Reply:
```suggestion
	`Land on the selected planet. If no planet is selected, automatically selects a planet to land on, or land on the planet you are directly on top of if traveling at low speeds. If a planet is already selected, double tapping will cycle to the next planet.`
```